roles/cs.php-fpm/tasks/001-daemon.yml

```diff
@@ -24,9 +24,9 @@
     line: >-
       {{ entry.type | default('d') }}
       {{ entry.path }}
-      {{ entry.mode | default('0755') }}
+      {{ entry.mode | default('0775') }}
       {{ entry.user | default('root') }}
-      {{ entry.group | default('root') }}
+      {{ entry.group | default('magento') }}
```

Comment:
This will also change permissions to run directory with is not intended behavior

It would be better to keep defaults here and overwrite settings on line 34 like this:
```yaml
  loop:
    - path: "{{ php_fpm_log_dir_path }}"
      mode: "775"
      group: "magento"
    - path: "{{ php_fpm_run_dir_path }}"
```